spikingjelly/activation_based/neuron/lif.py

```
@@ -591,77 +641,33 @@ def multi_step_forward(self, x_seq: torch.Tensor):
-                                x_seq, self.v, self.v_threshold, self.tau
-                            )
-                        )
-                    else:
-                        spike_seq, self.v = (
-                            self.jit_eval_multi_step_forward_soft_reset_decay_input(
-                                x_seq, self.v, self.v_threshold, self.tau
-                            )
-                        )
-                else:
+            # All backends: try Triton on GPU first (unified kernel covers all
+            # soft/hard reset × decay_input variants via tl.constexpr).
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)